runtime/parachains/src/configuration.rs

```diff
@@ -213,6 +220,11 @@ impl<BlockNumber: Default + From<u32>> Default for HostConfiguration<BlockNumber
 			hrmp_max_parachain_outbound_channels: Default::default(),
 			hrmp_max_parathread_outbound_channels: Default::default(),
 			hrmp_max_message_num_per_candidate: Default::default(),
+
+			//
+			// FOR REVIEW: what is a sensible value here?
+			//
+			ump_max_individual_weight: 10 * WEIGHT_PER_MILLIS,
```

Comment:
should probably be some percentage of the block weight. Something like 1% of the block weight does not sound unreasonable. That would be .02 seconds.

Reply:
Put `20 * WEIGHT_PER_MILLIS`. FWIW, this is 1/5 of the `ump_total_service_weight`.